### ISSUE TITLE
Bug/Unintended: Saving up 10km eggs until lvl 20 

### ISSUE THREAD

**Issue by CreativiTimothy** [NoxxDev/NecroBot#512]
According to the TheSilphRoad, https://www.reddit.com/r/TheSilphRoad/comments/4xdnzq/10km_egg_question/d6ejd6c

the level of the egg will be determined when you first OBTAINED the egg. If you got a egg at lvl5, and then you save it up until level 20-25 or something, it will still hatch a lvl5 pokemon. However, the bot waits all the way until lvl20 before hatching, which is pointless because it will still hatch at lvl5 or whatever lvl you got it at

**Comment by VBQL** [comment]
It is a value you could turn off if you want.

**Comment by CreativiTimothy** [comment]
@VBQL Really? If so, where is it in the config? I couldn't find it here:
    "UseIncenseConstantly": false,
    "UseEggIncubators": true,
    "UseEggIncubatorMinKm": 10,
    "UseLuckyEggConstantly": false,

**Comment by kirito-su** [comment]
Are you Affirmative on that a Held Egg comes out at its obtained level??

**Comment by CreativiTimothy** [comment]
@kirito-su TheSilphRoad did already confirm that the egg Pokemon is predetermined by the location of where the egg is from. For example, if you picked up an egg at a beach, and then you hatched it in a desert, it will still be a Magikarp, or whatever water Pokemon that spawns at the beach. The link I posted up there and how egg Pokemon is predetermined makes me pretty affirmative that it comes out at its obtained level. Not an extreme example, but I saw my 5km egg hatch a lvl 18 Pokemon when I was at level 20
